odoo/fields.py

```diff
@@ -2487,7 +2494,7 @@ def convert_to_read(self, value, record, use_name_get=True):
                 # Should not happen, unless the foreign key is missing.
                 return False
```

Comment:
I should probably convert this hardcoded False and also the (value.id, ...) tuple above.